internal/migrations/README.md

```diff
@@ -0,0 +1,35 @@
+#Migrations
```

Comment:
Some markdown viewers require a space between the `#` and the text to render a header properly.